src/Pages/SignIn/Google.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom';
+
+const withRouter = (Component) => {
+  const Wrapper = (props) => {
+    const navigate = useNavigate();
+    
+    return (
+      <Component
+        navigate={navigate}
+        {...props}
+        />
+    );
+  };
+  
+  return Wrapper;
+};
+
+class Google extends React.Component {
+
+  constructor(props) {
+    super(props)
+    window.customCallback = this.customCallback.bind(this)
+    this.customCallback=this.customCallback.bind(this);
+  }
+  
+  customCallback(response) {
+    localStorage.setItem("token", response.credential)
+    this.props.navigate('/dashboard')
+  }
+  
+    componentDidMount(){
+        const script = document.createElement('script');
+        script.id = 'google-script'
+        script.src = "https://accounts.google.com/gsi/client" ;
+        script.async = true;
+        script.defer = true;
+      
+        document.body.appendChild(script);
+    }
+
+    componentWillUnmount(){
+        const googleScript = document.getElementById('google-script')
+        document.body.removeChild(googleScript)
+    }
+ 
+  
+
+  render() {
+    return (
+      <div>
+        <div>click here to sign in with google!!!!!</div>
```

Comment:
Let's do a button here instead of text.

Reply:
ok. sounds good. I'm about to fix the merge conflict now and can do it. 
should I do ... ??
<div>
   <button> Sign In with Google!</button>
</div>

Reply:
all done. Messaging to resolve for merge